examples/cu_consolemon/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
```

Comment:
Should this live in examples? Seems like it'd fit better in a tools/ directory

Reply:
good call I think we need to reorg the repo. Also, now that I cleaned up the PID algorithm, I think we can have a little tree of drivers/algorithms/monitors as proper components. I will do that after merging this because another big PR is based upon this one.